SU2_CFD/src/iteration/CDiscAdjFluidIteration.cpp

```diff
@@ -343,8 +349,11 @@ void CDiscAdjFluidIteration::Iterate(COutput* output, CIntegration**** integrati
                                      CSolver***** solver, CNumerics****** numerics, CConfig** config,
                                      CSurfaceMovement** surface_movement, CVolumetricMovement*** volume_grid_movement,
                                      CFreeFormDefBox*** FFDBox, unsigned short iZone, unsigned short iInst) {
-  bool frozen_visc = config[iZone]->GetFrozen_Visc_Disc();
-  bool heat = config[iZone]->GetWeakly_Coupled_Heat();
+
+  SU2_OMP_PARALLEL_(if(solver[iZone][iInst][MESH_0][ADJFLOW_SOL]->GetHasHybridParallel())) {
+
+  const bool frozen_visc = config[iZone]->GetFrozen_Visc_Disc();
```

Comment:
This type of action (ExtractAdjoint_Variables) was made hybrid parallel for the disc adj fluid iteration and solver but not for the FEA iteration because each fluid iteration is usually lighter than one structural iteration (and so it would be more of a bottleneck to run the fluid part only by one thread).